modules/rostests/apitests/user32/ScrollBarRedraw.c

```diff
@@ -0,0 +1,727 @@
+/*
+ * PROJECT:     ReactOS api tests
```

Comment:
```suggestion
 * PROJECT:     ReactOS API tests
```

Reply:
Updated, thanks!  Do note however that a bunch of other tests have the word "api" lowercase as well, I happened to copy it from one of those.